tests/integration/pilot/istioctl_test.go

```diff
@@ -74,6 +74,33 @@ Next Step: Add related labels to the deployment to align with Istio's requiremen
 	removeFromMeshPodAOutput = `deployment .* updated successfully with Istio sidecar un-injected.`
 )
 
+func TestWait(t *testing.T) {
+	framework.NewTest(t).
+		Run(func(ctx framework.TestContext) {
+			ns := namespace.NewOrFail(t, ctx, namespace.Config{
+				Prefix: "default",
+				Inject: true,
+			})
+			ctx.ApplyConfigOrFail(t, ns.Name(), `
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: reviews
+spec:
+  gateways: [missing-gw]
+  hosts:
+  - reviews
+  http:
+  - route:
+    - destination: 
+        host: reviews
+`)
+			istioCtl := istioctl.NewOrFail(ctx, ctx, istioctl.Config{})
+			istioCtl.InvokeOrFail(t, []string{"x", "wait", "VirtualService", "reviews." + ns.Name()})
+			fmt.Print("end")
```

Comment:
code LGTM but not sure about this. At the very least should remove the print

Is this supposed to succeed or fail? since it has a missing gw

Reply:
This will succeed, as a missing gw does not impact distribution of config.

If Analysis were enabled, you would see a condition in the status field of PassedAnalysis=false, but distribution would still not be impacted.